demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
+1. Download the english language file.
+1. Add the translation keys in [en_translations](en_translations.properties)
+1. Upload the translation file. 
+1. Upload the contact forms using medic-conf `medic-conf upload-contact-forms`
```

Comment:
Is `--local` implied when you don't add it explicitly?!  that'd be awesome! if not, please add.

Reply:
I'll add it if `--local` is what we document. However, there are many options for instances. I feel like having `--local` in one place and not others could be confusing.  Especially since `--local` requires you to have env vars set that you don't really need to use medic-conf.

Reply:
added note about <instance>